projects/admin-core/src/lib/pages/search-index-page/search-index-page.component.html

```diff
@@ -1,6 +1,16 @@
-<div class="admin-page-wrapper" [ngClass]="className$ | async">
-  <div class="admin-tree">
-    <tm-admin-search-index-list></tm-admin-search-index-list>
-  </div>
-  <div class="admin-content" [ngClass]=><router-outlet></router-outlet></div>
-</div>
+
+@if(searchIndexPingResponse$ | async; as pingResponse) {
+  @if (pingResponse.success) {
+    <div class="admin-page-wrapper" [ngClass]="className$ | async">
+      <div class="admin-tree">
+        <tm-admin-search-index-list></tm-admin-search-index-list>
+      </div>
+      <div class="admin-content" [ngClass]=><router-outlet></router-outlet></div>
+    </div>
+  } @else {
+    <p>Search indexes are not available. This may be because Solr is not deployed or not operating correctly.
```

Comment:
Maybe nice to wrap this in `<div class="admin-page-wrapper"><div class="admin-content"></div></div>` for consistent layout (paddings etc.). And why not two `<p>` below each other instead of single `<p>` with <br> inside?

Also, you could wrap this in a `<tm-error-message [friendlyError]="true">` so you get the error message layout